netket/driver/vmc.py

```diff
@@ -48,23 +55,11 @@ def __init__(
             hamiltonian: The Hamiltonian of the system.
             optimizer: Determines how optimization steps are performed given the
                 bare energy gradient.
-            sr: Determines whether and how stochastic reconfiguration
-                is applied to the bare energy gradient before performing applying
-                the optimizer. If this parameter is not passed or None, SR is not used.
-            sr_restart: whever to restart the SR solver at every iteration, or use the
-                previous result to speed it up
-
-        Example:
-            Optimizing a 1D wavefunction with Variational Monte Carlo.
-
-            >>> import netket as nk
-            >>> hi = nk.hilbert.Spin(s=1 / 2, N=g.n_nodes)
-            >>> ha = nk.operator.Ising(hilbert=hi, graph=g, h=1.0)
-            >>> ma = nk.models.RBM(alpha=1, use_visible_bias=True, dtype=float)
-            >>> sa = nk.sampler.MetropolisLocal(hi, n_chains=16)
-            >>> op = nk.optimizer.Sgd(learning_rate=0.1)
-            >>> gs = nk.driver.VMC(ha, op, sa, ma, n_samples=1000, n_discard=50)
-            >>> gs.run(n_iter=300, out="test")
+            preconditioner: Determines which preconditioner to use for the loss gradient.
+                This must be a tuple of `(object, solver)` as documented in the section
+                `preconditioners` in the documentation. The standard preconditioner
+                included with NetKet is Stochastic Reconfiguration. By default, no
+                preconditioner is used and the bare gradient is passed to the optimizer.
```

Comment:
Sorry I was late to this party... This docstring doesn't match the actual behaviour, if you pass a tuple it errors out on line 132, could you fix it?

Reply:
Ah that was the first behaviour and i forgot to update the docstrings.
Yeah ill push an update in the next few days.